src/Student5/Ex4/Main.js

```diff
@@ -1,10 +1,28 @@
-import React from 'react'
-import './Main.css'
+import React from "react";
+import { Tweets } from "./Tweets";
+
+
+import "./Main.scss";
 
 export const Main = () => {
-  return (
-    <div>
-        Hello I'm student id: 3.
+  
+return (
+    <div className="Main">
+
+     
+
+      <div>Total Likes: 13 </div> 
```

Comment:
We cannot hardcode the value of 13 here.  The assignment says:

The Tweets component has [state](https://gitlab.com/mburolla/javascript-react-starter/-/blob/main/src/comps/Type9.js#L8) that keeps track of the total number of likes.